Tutorials/whatiswot/website/docs/wot/application-domains/smart-home-and-cities.md

```diff
@@ -13,56 +13,53 @@ sidebar_label: Smart Homes and Cities
-You can have direct remote access through gateway and routers,
-or interactions happen through an intermediary like a cloud application.
+> This helps with various activities within the home, such as controlling lights, adjusting room temperature, managing home appliances, and enhancing security using smartphones. These capabilities optimize energy and resource consumption.
 
-Services running in the cloud collect data from edge devices and provide the data to users of the smart home through edge devices such as phones or PCs.
+### Architecture patterns
 
-Multiple edge devices are connected to the gateways through local communication protocols such as KNX, ECHONET, ZigBee, DECT ULE, and Wi-SUN. Multiple gateways can exist in one home, with each supporting multiple local protocols.
+Now that we’ve covered the basics of smart homes, let’s take a closer look at the architecture patterns that enable these systems to work efficiently in a typical local network and remote-control scenario.
 
-![smart-home-connection](/img/10-Application-Domains/smart-home-connection.png)
+1. **Controllers:**
 
-## Industrial Applications
+A common pattern involves using a controller, locally and/or remotely, to interact with devices on the home network by consuming the Thing Description. Devices are represented by TDs and act as servers, responding to commands from client controllers (for example, smartphones) for actions such as reading sensor values or actuating devices.
 
-Industrial applications of WoT involve more complex use cases compared to smart homes. In factories, for instance, advanced monitoring of connected manufacturing equipment and products is essential, along with predictive maintenance to anticipate machine failures.
+![smart-home](/img/10-Application-Domains/smart-homes.png)
 
-![inside-of-factory](/img/10-Application-Domains/inside-of-factory.png)
+2. **Gateways:**
 
-Let's look at one smart factory example in detail. Here, different factory equipment is automated based on industrial protocols such as PROFINET, Modbus, OPC UA TSN, EtherCAT, or CAN. Since industrial environments like factories, have more strict requirements, there are multiple controllers that are connected to the manufacturing equipment.
+Gateways are another pattern commonly found in Smart Homes. They connect home networks to the internet, enabling remote management and monitoring.
 
-An industrial edge device collects data from various controllers and enables remote monitoring via a dashboard or analyzes it for preventative maintenance.
+A gateway can use the local TD of a device to provide web-level access management, allowing devices to be accessed and managed both locally and remotely over the web. In the local network, the device uses one protocol for control, while externally, another protocol is typically used to manage remote interactions. The gateways facilitate communication between local devices and external controllers, enhancing accessibility and control.
```

Comment:
The second sentence can be explained in more detail with a diagram. Modbus Thing <-> Gateway <-> HTTP Thing

Then saying that the TD stays the same except for protocol details. This can be a _tip_ as well